PKGBUILD

```diff
@@ -79,6 +78,16 @@ prepare() {
 
   cd "$srcdir/chromium-${_chromium_version}"
 
+  # Allow building against system libraries in official builds
+  sed -i 's/OFFICIAL_BUILD/GOOGLE_CHROME_BUILD/' \
+    tools/generate_shim_headers/generate_shim_headers.py
+
+  # https://crbug.com/893950
+  sed -i -e 's/\<xmlMalloc\>/malloc/' -e 's/\<xmlFree\>/free/' \
```

Comment:
This is already handled in `patches/inox-patchset/fix-cfi-failures-with-unbundled-libxml.patch`

Reply:
I'd like to reduce the amout of patches needed. That whole patch can be replaced with that one liner.